recipes/kafka-python-ng/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "kafka-python-ng" %}
+{% set version = "2.0.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "b447ce219097e27741a46c9d1ee5cb55f42974336277bf41a456bba7623e3078" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://github.com/wbarnha/{{ name }}/archive/v{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  noarch: python
+  script:
+    - "{{ PYTHON }} setup.py bdist_wheel --dist-dir=dist"
+    - "{{ PYTHON }} -m pip install --find-links=dist --no-deps --ignore-installed --no-cache-dir -vvv {{ name }}"
+
+
+requirements:
+  build:
+    - python
+    - git
+    - pip
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - wheel
+
+  run:
+    - python
```

```suggestion
    - python >= 3.8
```

Looks like this is a requirement for their checker script. Looks like it also needs a similar line under a `host:` section.